platform/viewer/public/config/default.js

```diff
@@ -18,7 +18,7 @@ window.config = {
   maxNumRequests: {
     interaction: 100,
     thumbnail: 75,
-    prefetch: 10,
+    prefetch: 25,
```

Comment:
The prefetch number is heavily dependent on protocol - running http/2 or /3 allows for high prefetch numbers, but a prefetch of 25 is too high for /1.1 or /1.0.  The issue is that it starts blocking the fetch queue.  I think I would add a comment here explaining a bit of that - I would also try as high as 100 and see if it makes a difference.  If you consider a latency of 200 ms, for a fetch size of 200k, then the transfer time probably on the 20 ms time frame, and you want to completely consume all the latency time with transfer time, so that means in a perfect overlap situation, you need 20/200 = 10 prefetch, which in a practical world with random transfer times, you want 2-3 times that amount, so 25 is a good practical amount.